pages/docs/couple-your-code/couple-your-code-initializing-coupling-data.md

```diff
@@ -5,45 +5,27 @@ keywords: api, adapter, initialization, coupling scheme, restart
 summary: "As default values, preCICE assumes that all coupling variables are zero initially. For fluid-structure interaction, for example, this means that the structure is in its reference state. Sometimes, you want to change this behavior – for instance, you may want to restart your simulation."
 ---
 
-For initializing coupling data, you can add the following **optional** method:
+By default preCICE assumes that all coupling variables are zero initially. If you want to provide non-zero initial values, you can write data before calling `initialize()`. This data will then be used as initial data. To check whether initial data is required, you can use the following function:
 
 ```cpp
-void initializeData();
+bool requiresInitialData()
```

Comment:
This removal does not really belong to this documentation update, because it is triggered by the update of the action interface. But I did it here anyway.